examples/showcase/xkcd.py

```diff
@@ -14,8 +14,8 @@
     # Based on "Stove Ownership" from XKCD by Randall Munroe
     # https://xkcd.com/418/
 
-    fig = plt.figure()
-    ax = fig.add_axes((0.1, 0.2, 0.8, 0.7))
+    fig, ax = plt.subplots()
+    ax.set_position((0.1, 0.2, 0.8, 0.7))
```

Comment:
`add_axes()` is specifically designed for adding axes at arbitrary locations. IMHO it's the right method to use here. Creating a figure and then creating an axes at a specific position is conceptually simpler than creating a figure and an axes in the default layout and then moving the existing axes raound.

Reply:
`subplots` sets up a gridspec for the underlying axes, whereas if you are just going to move it that gridspec is not needed.  So there is some advantage to `add_axes` here.

Reply:
Hi, no problem I can revert it to the original version, but I want to make sure I understand - you suggest to keep it the same way it was in the beginning? or some other method using both `subplots` and `add_axes` (if it's feasible, didn't already thought about it to details).

Reply:
That example was already correct as it was originally. Please revert your change.